install.bat

```diff
@@ -7,12 +7,25 @@ SET "yenpath=%userprofile%\.yen\bin"
 REM Create the .yen\bin directory if it doesn't exist
 mkdir "%yenpath%" 2>nul
 
+REM Function to download using curl or wget
+:download
+if exist "%SystemRoot%\System32\curl.exe" (
+    curl -SL --progress-bar "%1" --output "%2"
+) else if exist "%SystemRoot%\System32\wget.exe" (
+    wget "%1" -O "%2"
+) else (
+    echo Neither curl nor wget is installed. Please install one of them and try again.
+    exit /b 1
+)
+goto :eof
```

Comment:
Where's eof defined?

We don't want to jump to end of file after this, the PATH addition code is yet to be run.

Reply:
@tusharsadhwani I've fix this :-)